calicoctl/resourcemgr/resourcemgr.go

```diff
@@ -25,17 +25,17 @@ import (
 	"strings"
 	"time"
 
+	"github.com/projectcalico/calicoctl/calicoctl/commands/argutils"
```

Comment:
nit:  I think it's nice to organize imports in 3 groups.   Built-ins, External, projectcalico

Reply:
shouldn't stop this going in though

Reply:
Ah, I have my IDE setup to just run `go imports` on all modified files I think, do we need this before merging?